### PR TITLE
#9 | Add Cluster Health API 

### PR DESCRIPTION
**Pull request by m4ver1k**
Fixes https://github.com/esctl/esctl/issues/9

### CODE REVIEW

File: pkg/es/client.go [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+	Status      string
+}
+
+type mockTransport struct {
+	roundTripFunc func(*http.Request) (*http.Response, error)
+}
+
+func (t *mockTransport) RoundTrip(req *http.Request) (*http.Response, error) {
+	return t.roundTripFunc(req)
+}
```
Comment (jijotj) [inline pkg/es/client.go:62]: This should be in the test file as it's only used there.

File: pkg/es/client_test.go [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+		"number_of_nodes": 3,
+		"number_of_pending_tasks": 0,
+		"relocating_shards": 0,
+		"status": "green",
+		"task_max_waiting_in_queue_millis": 0,
+		"timed_out": false,
+		"unassigned_shards": 0
+	}
+	`
+	client := ElasticSearchClient{}
```
Comment (jijotj) [inline pkg/es/client_test.go:58]: Instantiate using New() in tests for better coverage? Also serves as a documentation that `ElasticSearchClient` should be instantiated using New() only. We can make the struct private if there's no need for it to be exposed outside.

File: pkg/es/client.go (hunk 1 above)
Comment (m4ver1k) [reply pkg/es/client.go:62]: will move it a `client_mock.go` will be needed in multiple test files.

File: pkg/es/client_test.go (hunk 2 above)
Comment (m4ver1k) [reply pkg/es/client_test.go:58]: agree will make the changes.

File: pkg/es/client.go (hunk 1 above)
Comment (m4ver1k) [reply pkg/es/client.go]: pushed changes @jijotj

File: pkg/es/client_test.go (hunk 2 above)
Comment (m4ver1k) [reply pkg/es/client_test.go]: pushed the changes

**Review by jijotj — APPROVED** [review]

**Pull request merged**